mtt/calibration/jets.py

```diff
@@ -42,3 +44,73 @@ def jet_energy_init(self: Calibrator) -> None:
     else:
         self.uses |= {jec_nominal}
         self.produces |= {jec_nominal}
+
+
+@calibrator(
+    uses={
+        "Electron.pt", "Electron.eta", "Electron.phi", "Electron.mass", "nElectron",
+        "Muon.pt", "Muon.eta", "Muon.phi", "Muon.mass", "nMuon",
+        "Jet.pt", "Jet.eta", "Jet.phi", "Jet.mass", "Jet.rawFactor", "nJet",
+        "Jet.muonIdx1", "Jet.muonIdx2", "Jet.electronIdx1", "Jet.electronIdx2",
+        attach_coffea_behavior,
+    }
```

Comment:
Looks like a comma is missing here:

```suggestion
    },
```